docs/handbook/the-shim.md

```diff
@@ -48,9 +48,9 @@ import { setup, disconnect } from 'twind/shim'
 
 <!-- > [live and interactive shim demo 🚀 ](https://esm.codes/#aW1wb3J0ICdodHRwczovL2Nkbi5za3lwYWNrLmRldi90d2luZC9zaGltJwoKZG9jdW1lbnQuYm9keS5pbm5lckhUTUwgPSBgCiAgPG1haW4gY2xhc3M9Imgtc2NyZWVuIGJnLXB1cnBsZS00MDAgZmxleCBpdGVtcy1jZW50ZXIganVzdGlmeS1jZW50ZXIiPgogICAgPGgxIGNsYXNzPSJmb250LWJvbGQgdGV4dChjZW50ZXIgNXhsIHdoaXRlIHNtOmdyYXktODAwIG1kOnBpbmstNzAwKSI+CiAgICAgIFRoaXMgaXMgVHdpbmQhCiAgICA8L2gxPgogIDwvbWFpbj4KYA==) -->
 
-## Prevent FOUC
+## Prevent FOUC / CLS
 
-To prevent FOUC (_flash of unstyled content_), set the `hidden` attribute on the target element. `twind/shim` will remove it once all styles have been generated.
+To prevent FOUC (_flash of unstyled content_) or more probably known as Cumulative Layout Shift, set the `hidden` attribute on the target element. `twind/shim` will remove it once all styles have been generated.
```

Comment:
This is incorrect.

Firstly, you wouldn't prevent **cumulative** layout shifting (emphasis mine); you'd prevent layout shifting. The C of CLS is specifically the measurement, not the effect.

Secondly, FOUC and CLS are two very different things. FOUC is not known as CLS. An FOUC _could_ lead to layout shifting, but that's not necessarily guaranteed, nor is that the only source. For example, an image that has not been given proper height & width properties would cause shifting of content upon loading in.

Reply:
agreeing with you, but yea layout shift can be the name here then; sorry because I used the same name from lighthouse tools.

Reply:
should be fixed hopefully?

Reply:
I wouldn't abbreviate to "LS", as that's not a common abbreviation, and the text is still wrong:

> or more probably known as Layout Shift

FOUC is not "more probably known as layout shift". They're two entirely different things. Again, FOUC can lead to shifting, but so can a number of other things.

```diff
- To prevent FOUC (_flash of unstyled content_) or more probably known as Layout Shift, set the `hidden` attribute on the target element. `twind/shim` will remove it once all styles have been generated.
+ To prevent FOUC (_flash of unstyled content_) and layout shifting, set the `hidden` attribute on the target element. `twind/shim` will remove it once all styles have been generated.
```

That's how I'd word it, anyways.